integration/anchor/programs/anchor/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+use anchor_lang::prelude::*;
+
+declare_id!("z7FbDfQDfucxJz5o8jrGLgvSbdoeSqX5VrxBb5TVjHq");
+
+/// Anchor example for testing with Solang.
+/// This doc comment exist for testing metadata doc comments
+#[program]
+pub mod anchor {
```

Comment:
Should we test events too?

Reply:
This PR is for calling Anchor contracts. We're not generating IDL files yet for Solidity. Testing emitting events from Solidity is not something we can test; testing emiting events from Anchor -- I am not sure that is worth testing at this point.